utils/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+//! The utils crate
+//!
+//! This crate provides a place in which we can temporary put
+//! implementations of tools that are of common use. Some examples 
+//! are testing tools (unit tests) and also conversion tools.
+//!
+//! It can be seen as an incubator of tools. It will also avoid
+//! entangling the dependency tree among contracts for things are
+//! note related with them.
+
+#![no_std]
+
+use soroban_sdk::{Address, BytesN, TryFromVal, Val};
+
+pub fn address_to_bytes_n32(addr: &Address) -> BytesN<32> {
```

Comment:
Yeah, i dont like this either. Because this also means whoever is deploying the contract needs to be aware of this "black magic".

Sadly, i think such functions `contract_id` are not public and are only available in `testutils` . But i really think it would be very useful to have them available. 

Looks like this is something that is resonating in the community: https://discordapp.com/channels/897514728459468821/1136646522910552114 . We could expose our use case there. Wdyt ?

An alternative to this, would be to add a `new_curator_addr: Some(Address)` to the `proposal` struct in the voting contract. But the downside of this is that we would need to parametrize that in all interfaces of the voting contract. And i consider it a bit excessive.